packages/libfmt.rb

```diff
@@ -0,0 +1,19 @@
+require 'package'
+
+class Libfmt < Package
+  description 'A modern formatting library'
+  homepage 'https://fmt.dev'
+  version '7.1.3'
+  compatibility 'all'
+  source_url 'https://github.com/fmtlib/fmt/releases/download/7.1.3/fmt-7.1.3.zip'
+  source_sha256 '5d98c504d0205f912e22449ecdea776b78ce0bb096927334f80781e720084c9f'
+
+  def self.build
+    system "cmake #{CREW_CMAKE_OPTIONS} ."
```

Comment:
Typically, when building using cmake, a `build` directory is needed.  Did you follow the build instructions?

Reply:
cmake didn't respond with any errors or warnings when I built this

Reply:
That still doesn't answer my question. :)

Reply:
Yes, and they're (here)[https://fmt.dev/latest/usage.html#building-the-library]. Although the instructions say to make a `build` directory, they also say right above "The typical workflow starts with:", implying that it isn't necessary. You're welcome to add a build dir to this package script should you like tho :)

Reply:
The idea of the build directory is to isolate the build from the source.  That way the package doesn't end up containing extra files that are unnecessary.  I think this is a very important concept to understand.

Reply:
I didn't know that, thank you! A fix is on its way.